tendermint/src/hash.rs

```diff
@@ -28,6 +28,12 @@ pub enum Hash {
     Null,
 }
 
+impl Default for Hash {
+    fn default() -> Hash {
+        Hash::Null
```

Comment:
Does this actually make sense? Honestly I think this whole concept should be replaced with `Option<Hash>` instead...

Reply:
Agree, but that probably should belong to another PR?